docs/how-to-guides/construct-test-data.md

```diff
@@ -0,0 +1,44 @@
+# How to construct test data based on documents
+This guide will help to construct test data based on the provided documents.
+The test data construction process contains three steps:
+- Split documents to smaller trunks.
+- Based on each document trunk generate a test data containing `question`, `answer`, `context` and `question_type`.
+By `question_type`, the given flow sample would evolve the simple question into more diverse question types like reasoning and conditional.
+- Collect all the test data and remove empty values.
+
+## Data preprocess
+### Local
+#### Prerequisites
+Enter `test_data_gen_local` folder, run below command to install required packages.
```

Comment:
install dependencies looks unrelated to enter folder?

Reply:
Enter `test_data_gen_local` folder is not clear since user need to search where the test_data_gen_local is.